Booking view completed

jeenakuriak opened the pull request:
Views for booking implemented. corrected the name of bookings controller

awinabi commented on app/views/bookings/new.html.erb:
```diff
@@ -0,0 +1,13 @@
+  <p>
+    <%= f.submit %>
+  </p>
+<% end %>
```
Please configure sublime to add newline at the end of file .- https://robots.thoughtbot.com/no-newline-at-end-of-file

You can use this config: https://github.com/awinabi/dotfiles/blob/master/.config/sublime-text-3/Packages/User/Preferences.sublime-settings

awinabi commented on app/views/bookings/new.html.erb:
```diff
@@ -0,0 +1,13 @@
+<%= form_for @booking do |f| %>
+  <p>
```
Since the `new.html.erb` & `edit.html.erb` uses the same form you can create a partial _form.htm.erb and use it in both these views.

Rails will add the appropriate url for a new record / edit. Refer: http://apidock.com/rails/ActionView/Helpers/FormHelper/form_for (Section - Resource Oriented Style)

awinabi commented on app/controllers/bookings_controller.rb:
```diff
@@ -28,7 +28,7 @@ def edit
 
   def update
     if @booking.update_attributes(post_params_edit) 
```
Is `post_params_edit` defined?

awinabi commented on app/controllers/bookings_controller.rb:
```diff
@@ -28,7 +28,7 @@ def edit
+      flash[:notice] = "Successfully edited #{@booking.status}"
       redirect_to booking_index_path   
     else
       flash[:notice] = "Error: Please check your inputs."
```
You can use `@booking.errors` in the UI to show which specific errors occured.

awinabi commented on app/controllers/bookings_controller.rb:
```diff
@@ -1,5 +1,5 @@
+  load_and_authorize_resource except: [:create]
   
   def index
     @booking = Booking.all
```
Please verify if there is already a variable `@bookings` introduced by  `load_and_authorize_resource`

jeenakuriak commented:
1. added partials for new and edit
2. changed the booking controller (load and authorize)
3.add address and pincode fields to booking table
4. set default status as pending in bookings table

awinabi commented:
Looks good! :+1:
